Chamasma/Presentation/Sources/Presentation/Examples/Meal+Examples.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  Menu+Examples.swift
+//  
+//
+//  Created by Mariam Ormotsadze on 23.04.23.
+//
+
+import Domain
+import DesignSystem
+
+// MARK: - Example
+public extension Meal {
+    enum examples: CaseIterable {
```

Comment:
Enum should start with upper case, and be in singular form
```suggestion
    enum Example: CaseIterable {
```